<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="startgame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="t_window">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>700</y>
      <width>1600</width>
      <height>300</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="next">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>740</y>
      <width>1541</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>游明朝</family>
      <pointsize>40</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="place_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>810</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="place_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="place_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="place_4">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>60</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="place_5">
    <property name="geometry">
     <rect>
      <x>1341</x>
      <y>800</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="next_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</normaloff>../build-Finalprojec-Desktop_Qt_5_15_1_MinGW_64_bit-Debug/images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="start">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>300</y>
      <width>1000</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/white.png</normaloff>images/white.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="end">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>630</y>
      <width>1000</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/white.png</normaloff>images/white.png</iconset>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>startgame</zorder>
   <zorder>t_window</zorder>
   <zorder>label</zorder>
   <zorder>next</zorder>
   <zorder>place_1</zorder>
   <zorder>place_2</zorder>
   <zorder>place_3</zorder>
   <zorder>place_4</zorder>
   <zorder>place_5</zorder>
   <zorder>next_2</zorder>
   <zorder>start</zorder>
   <zorder>end</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>end</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1436</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>1513</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
